src/VisualStudio/Core/Test/CodeModel/AbstractCodeElementTests`1.vb

```diff
@@ -151,6 +151,10 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.CodeModel
         Protected MustOverride Function GetName(codeElement As TCodeElement) As String
         Protected MustOverride Function GetNameSetter(codeElement As TCodeElement) As Action(Of String)
 
+        Protected Overridable Function GetNamespace(codeElement As TCodeElement) As EnvDTE.CodeNamespace
```

Comment:
`Namespace` is an `EnvDTE80.CodeClass2` property but not `CodeElement` property.